linux_pipeline/Jenkinsfile_ubuntu_azure_kernel_validation

```diff
@@ -0,0 +1,202 @@
+#!/usr/bin/env groovy
```

Comment:
in the commit message, use ubuntu-azure-kernel: instead of [ubuntu-azure-kernel], we decided to keep the linux kernel commit message way